raiden/tests/integration/cli/conftest.py

```diff
@@ -107,6 +107,8 @@ def cli_args(request, tmpdir, raiden_testchain, removed_args, changed_args, envi
             initial_args["service_registry_contract_address"],
             "--environment-type",
             environment_type,
+            "--one-to-n-contract-address",
```

Comment:
Is this necessary? The loop below should do that anyway if it's in `initial_args`

Reply:
Good point, removed the unnecessary ones.